recipes/umnn/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "umnn" %}
+{% set version = "1.71" %}
+{% set python_min = "3.7" %}
```

Comment:
Be aware that conda-forge min supported Python is 3.9.

Reply:
Thanks for the fast merge, @ocefpaf!

Indeed, that's why I set the `python_min` here to try to be as close to PyPI as could be supported. My understanding (please correct me if I'm wrong here) is that the current global pins (so supported versions) are [Python 3.9 to 3.12](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml#L826-L841) and that these are the default supported versions, but, if you wanted to go outside those supported defauts that's still okay for `noarch: python` packages as long as you don't expect any support there. Is that correct?

Reply:
It is correct. The warning is mostly "if this breaks, it is unlikely we'll supported it." (It should work though.)

Reply:
Great and thanks very much for the info! I really appreciate the work that you and the other core team members do to keep everything running so smoothly.